### PR TITLE
Hidden content in code blocks

### PR DESCRIPTION
**Pull request by GeorgePlukov**

### CODE REVIEW

File: pages/assets/demo-snippet.js [hunk 1]
```diff
@@ -112,20 +115,24 @@ class ComponentCatalogDemoSnippetWrapper extends LitElement {
 		return parseImports(this._demoSnippet);
 	}
 
+	get strippedCode() {
```
Comment (GeorgePlukov) [inline pages/assets/demo-snippet.js:118]: Not super happy with this name

File: pages/assets/demo-snippet.js [hunk 2]
```diff
@@ -10,6 +10,9 @@ import { ifDefined } from 'lit-html/directives/if-defined.js';
 import { themeStyles } from './code-style.js';
 import { unsafeHTML } from 'lit-html/directives/unsafe-html';
 
+// eslint-disable-next-line no-control-regex
```
Comment (GeorgePlukov) [inline pages/assets/demo-snippet.js:13]: This rule is disabled to allow us to search for hidden/escaped characters in this case the option `\n` that we want to support

**Comment by GeorgePlukov** [comment]
Taking a look at the failing tests

File: pages/assets/demo-snippet.js (hunk 1 above)
Comment (margaree) [reply pages/assets/demo-snippet.js:118]: Hmm.. demoCode? preferredCode?

**Review by margaree — APPROVED** [review]

**Pull request merged**